source/server/config_validation/server.h

```diff
@@ -154,8 +154,9 @@ class ValidationInstance : Logger::Loggable<Logger::Id::main>,
   AccessLog::AccessLogManagerImpl access_log_manager_;
   std::unique_ptr<Upstream::ValidationClusterManagerFactory> cluster_manager_factory_;
   InitManagerImpl init_manager_;
-  ListenerManagerImpl listener_manager_;
```

Comment:
curious, why not leave this as-is and just create/declare  the overload manager before it?

Reply:
The overload manager constructor takes the bootstrap proto which isn't available until the initialize method, so I had to a little code shuffling here.